developersociety/commonslibrary
# Make editable footer logos

# Sources
https://trello.com/c/RQtkp4f6/21-fix-the-logos-of-partner-orgs-if-possible-make-it-so-they-can-edit-this-themselves

# Description
This PR makes it possible to toggle on/off displaying each partner org logo in the footer of the site, by ticking or unticking a box in the Organisations Directory in admin. 

# Setup instructions
`make django-dev-createsuperuser`
`./manage.py migrate`

# How to test
- Go to http://127.0.0.1:8000/admin/directory/organisation/ and click on any partner org that is already being displayed in the footer
- Untick the 'show logo on footer' box (located below Logo) and save changes
- Check that the logo has disappeared from the footer

- Add a new organisation and a logo in http://127.0.0.1:8000/admin/directory/organisation/ (you can download one of the existing logos that are being displayed, such as http://127.0.0.1:8000/admin/directory/organisation/2/change/)
- Save it, and check that the new logo appears in the footer

For comprehensive guidelines on code review at DEV, see here: https://www.notion.so/developersociety/Code-Review-f804361f9b6b4859bdcd69d7704168e5.

# Pre-review checklist
